geonode/layers/templates/layers/layer_detail.html

```diff
@@ -355,7 +355,7 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Download Layer" %}</h4>
                                           {% endfor %}
                                         {% elif resource.attributes %}
                                           {% for attribute in resource.attributes %}
-                                            <li><a href="#" id={{ attribute }} data-type='string' data-schema='no'>{{ attribute }}</a></li>
+                                            <li><a href="#" id={{ attribute.attribute }} data-type='string' data-schema='no'>{{ attribute }}</a></li>
```

Comment:
@petrus7 are you sure attribute is an object here? How can it be if it's also used as a text?